.github/workflows/smoke.yml

```diff
@@ -0,0 +1,119 @@
+name: Smoke Test
+
+on:
+  workflow_dispatch:
+  pull_request: # TODO Temporary, but maybe we might keep it
+
+jobs:
+  test:
+    name: ${{ matrix.driver.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        driver:
+          - name: BrowserKit (http client)
+            repo: https://github.com/minkphp/MinkBrowserKitDriver.git
+            php: 7.2
+            setup: |
+              mkdir ./logs
```

Comment:
You doing this command in each matrix element. Maybe you can move it to a shared place.